archivist/cmds/usage/run.py

```diff
@@ -0,0 +1,153 @@
+"""Run RKVST usage summary script."""
+
+from contextlib import suppress
+from logging import getLogger
+from sys import exit as sys_exit
+
+from os import path, makedirs, remove, getcwd
+from json import dump as json_dump
+from datetime import datetime
+from shutil import move, make_archive
+
+from archivist.dictmerge import (
+    assets_ext_attr,
+    attachment_identities_assets,
+    events_ext_attr,
+    attachment_identities_events,
+    _deepmerge,
+)
+
+LOGGER = getLogger(__name__)
+
+
+def run(arch, args):
+    """Run diagnostic script."""
+    props = {"confirmation_status": "CONFIRMED"}
+    attrs = {}  # attributes can be added to filer by name, type, etc.
+
+    # Total Number of Assets
+    assets = list(arch.assets.list(props=props, attrs=attrs))
+
+    # Assets with Extended Attributes
+    extended_attributes_asset = assets_ext_attr(assets)
+
+    # Assets with Associated Location
+    assets_with_location = list(
+        arch.assets.list(
```

Comment:
This executes the API to get the assets with a filter  but all assets are already available in the assets variable.
it would be more efficient to simply apply a list comprehension to assets?

Reply:
this list comprehension could then be a separate function in dictmerge.py and unittest scould be added

Reply:
Changed to single list comprehension and unit tests added